stable/mysql/values.yaml

```diff
@@ -55,3 +55,7 @@ configurationFiles:
 #  mysql.cnf: |-
 #    [mysqld]
 #    skip-name-resolve
+
+service:
```

Comment:
It might be useful to document the different types of services or where someone can learn what the different types of services are.

Reply:
Addressed.